packages/components/src/Text/Text.tsx

```diff
@@ -16,7 +16,7 @@ import { colors, IColor } from '../colors'
 import styled from 'styled-components'
 import type { Property } from 'csstype'
 
-type Element = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'p' | 'span'
+type Element = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'p' | 'span' | 'label'
```

Comment:
I think we should either remove this `label` from here, or add a way for the `htmlFor` to be handled in the component. Probably just adding `{ htmlFor?: string }` to `ITextProps` would be enough

Reply:
@naftis I need to also style the position of the label. In that case, the code will be
`          <Text htmlFor={id}>
          <Label >{label}</Label>
          </Text>`

Reply:
> * cursor: pointer

Okay